readthedocs/rtd_tests/tests/test_footer.py

```diff
@@ -235,11 +240,12 @@ class TestFooterPerformance(APITestCase):
 
     # The expected number of queries for generating the footer
     # This shouldn't increase unless we modify the footer API
-    EXPECTED_QUERIES = 9
+    EXPECTED_QUERIES = 13
```

Comment:
Hmm, this is less than ideal. Is this an additional query for each version?

Reply:
The explanation is the commit

> We were testing with non-build versions.
> But in production is more common to hit this with build versions.

Reply:
That number is the "normal" number of queries, we were testing for a no common case where the version wasn't built

Reply:
Ah I see, it's in the version_compare data. I wonder if we should be caching this, seems simple enough to cache if it's a lot of our footer queries.

Reply:
I will _never_ look at commit messages in PR's, so it's probably better to put it in the PR description.